webdriver/support/fixtures.py

```diff
@@ -142,3 +144,41 @@ def inner(path, query="", fragment=""):
                                   fragment))
         return rv
     return inner
+
+def create_dialog(session):
+    """Create a dialog (one of "alert", "prompt", or "confirm") and provide a
+    function to validate that the dialog has been "handled" (either accepted or
+    dismissed) by returning some value."""
+
+    def create_dialog(dialog_type, text=None, result_var=None):
```

Comment:
I guess I don’t care enough about this at the moment to start another discussion about this form of API. It’s good enough for now and better than the status quo.